Scrapers/VenueExplorer.py

```diff
@@ -1 +1,130 @@
-# TODO: Write scraping script here
\ No newline at end of file
+from bs4 import BeautifulSoup
+from requests import get
+import VenueClass
+import json
+import os
+
+allVenues = []  # Array to store all Venues of type VenueClass
+
+def extractVenue(venue):
+
+    # extract title
+    title_html = venue.find('div', class_='event-name')
+    title = ''
+    if title_html:
+        title = title_html.text
+
+    # extract tags
+    venue_tags = venue.find('b', class_='t-holder').text.split(",")
+    tags = []
+    for venue_tag in venue_tags:
+        tags.append(venue_tag.strip())
+
+    # extract link into the venue
+    venue_link = venue.find_all('a')[1]['href']
+    response = get(venue_link)
+    indiv_venue = BeautifulSoup(response.text, 'html.parser')
+
+    # extract image links
+    venue_images = indiv_venue.find_all('a', class_='example-image-link')
+    images = []
+    for venue_image in venue_images:
+        image = 'venuexplorer.com.sg' + str(venue_image['href'])
```

Comment:
Line 32 should be changed to 

**image = 'https://www.venuexplorer.com.sg' + str(venue_image['href'])**

so that it is a valid link.